demo/src/Components/Pagination/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+
+const Pagination = () => (
+  <section className="o-section-md">
+    <div className="o-site-wrap o-site-wrap--padding">
+      <h4>Pagination</h4>
+      <nav className="c-pagination" aria-label="Page navigation">
+        <ul className="c-pagination-list o-inline-list">
```

Comment:
`<ol>`

Reply:
Also are we not using `o-inline-list--spaced-md` ?  I assume the spacing on your screenshot is caused by inline-block word-spacing which gets lost when you have minified markup like when server-rendering React.

Reply:
done